_resources/port1.0/group/cmake-1.1.tcl

```diff
@@ -123,66 +303,55 @@ default configure.post_args {${worksrcpath}}
 # TODO: Handle the Java-specific configure.classpath variable.
 
 pre-configure {
+    # create the build directory:
+    file mkdir ${configure.dir}
+
     # The environment variable CPPFLAGS is not considered by CMake.
     # (CMake upstream ticket #12928 "CMake silently ignores CPPFLAGS"
     # <https://www.cmake.org/Bug/view.php?id=12928>).
     #
     # But adding -I${prefix}/include to CFLAGS/CXXFLAGS is a bad idea.
     # If any other flags are needed, we need to add them.
 
-    # In addition, CMake provides build-type-specific flags for Release (-O3
-    # -DNDEBUG), Debug (-g), MinSizeRel (-Os -DNDEBUG), and RelWithDebInfo
-    # (-O2 -g -DNDEBUG). If the configure.optflags have been set (-Os by
-    # default), we have to remove the optimization flags from the concerned
-    # Release build type so that configure.optflags gets honored (Debug used
-    # by the +debug variant does not set optimization flags by default).
+    # In addition, CMake provides build-type-specific flags for
+    # Release (-O3 -DNDEBUG), Debug (-g), MinSizeRel (-Os -DNDEBUG), and
+    # RelWithDebInfo (-O2 -g -DNDEBUG). If the configure.optflags have been
+    # set (-Os by default), we have to remove the optimisation flags from
+    # the concerned Release build type so that configure.optflags
+    # gets honored (Debug used by the +debug variant does not set
+    # optimisation flags by default).
     # NB: more recent CMake versions (>=3?) no longer take the env. variables into
     # account, and thus require explicit use of ${configure.c*flags} below:
-#     if {${configure.optflags} ne ""} {
-#         configure.args-append   -DCMAKE_C_FLAGS="-DNDEBUG ${configure.cflags}" \
-#                                 -DCMAKE_CXX_FLAGS="-DNDEBUG ${configure.cxxflags}"
-#     }
```

Comment:
If you delete this, the comment above ending with a colon no longer makes sense. I guess the two lines should be removed as well?

Reply:
I've reworded the comment section.